overlord/devicestate/handlers_install.go

```diff
@@ -197,13 +209,51 @@ func (m *DeviceManager) doSetupRunSystem(t *state.Task, _ *tomb.Tomb) error {
 		return fmt.Errorf("cannot make run system bootable: %v", err)
 	}
 
+	if useEncryption {
```

Comment:
we can/should do this earlier I think, inside the if trustedInstallObserver != nil